pyproject.toml

```diff
@@ -15,11 +15,14 @@ repository="https://github.com/sicpa-dlab/aries-acapy-plugin-http-uniresolver"
 
 [tool.poetry.dependencies]
 python = "^3.6.9"
-aries-cloudagent = {git = "https://github.com/hyperledger/aries-cloudagent-python", rev = "main"}
-PyYAML = "~=5.4"
 ecdsa = "~=0.16.1"
 
 
+[tool.poetry.dependencies.aries-cloudagent]
+    git = "https://github.com/hyperledger/aries-cloudagent-python"
+    rev = "main"
```

Comment:
This can be reverted back to the shorthand toml table:
```
aries-cloudagent = {git = "https://github.com/hyperledger/aries-cloudagent-python", rev = "main"}
```